website/static/js/pages/login-page.js

```diff
@@ -20,6 +20,17 @@ var activateToggleBox = function () {
     }
 };
 
+$(document).ready(function() {
+    var self = this;
+    self.campaign = $('#campaign').val();
```

Comment:
My previous comment disappeared, but I suggested that you refactor this to use data-binding rather than mixing Knockout, jQuery, and Mako. Also, why are you assigning to `self`?